Functions/CreateCommand.py

```diff
@@ -0,0 +1,20 @@
+from command_factory import *
+from ICommand import ICommand
+import json
+import os
+import keys
+
+class CreateCommand(ICommand):
+    def __init__(self, schema):
+        self.schema = schema
+
+    def execute(self):
+        if self.schema is None:
+            return "Schema not found"
+        data = json.load(open(self.schema, 'r'))
```

Comment:
I would rename schema to schema_path. and add it in constructor 
self.schema = json.load(open(self.schema_url, 'r'))